content/news/2019-12-03-mapstruct-and-quarkus.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "MapStruct and Quarkus - a match made in heaven?"
+author: Christian Bandowski
+date: "2019-12-03"
+tags: [news, examples]
+---
+
+This year is already over, but it was started with something new that came up in the Java world: [Quarkus](https://quarkus.io/). You may already have heard about it, if not, don’t worry, I will quickly summarize what it is.
+
+_tl;dr: Take a look at the README in our [examples repository](https://github.com/mapstruct/mapstruct-examples/tree/master/mapstruct-quarkus)._
```

Comment:
I don't think a reference to another readme is a good TL;DR :) I'd suggest to either omit it or add a summarizing sentence of what the post is about.